src/docs/examples/accordion-swe2.html

```diff
@@ -377,6 +377,306 @@ <h4>Related services</h4>
+                  </li>
+                  <li><input id="has-qbuild-been-contacted-no" name="has-qbuild-been-contacted"
+                             value="No" required="required" type="radio" /><label
+                    for="has-qbuild-been-contacted-no">No</label></li>
+                </ol>
+              </fieldset>
+            </li>
+
+            <li id="qbuild-information" class="section">
+              <fieldset>
+                <legend>
+                  <span class="h2">Building and Asset Services information</span>
+                </legend>
+                <ol class="questions">
+                  <li>
+                    <label for="qbuild-office-notified">
+                      <span class="label">Building and Asset Services office notified</span>
+                      <abbr class="required" title="(required)">*</abbr>
+                      <small class="hint">Example: Cannon Hill</small>
+                    </label>
+                    <input id="qbuild-office-notified" name="qbuild-office-notified" size="30"
+                           required="required" type="text" />
+                  </li>
+                  <li>
+                    <label for="staff-member-name">
+                      <span class="label">Building and Asset Services staff member name</span>
+                      <abbr class="required" title="(required)">*</abbr>
+                    </label>
+                    <input id="staff-member-name" name="staff-member-name" size="30"
+                           required="required" type="text" />
+                  </li>
+                </ol>
+              </fieldset>
+            </li>
+
+            <li id="lodgment-details">
+              <fieldset>
+                <legend>
+                  <span class="h2">Lodgement details</span>
+                </legend>
+                <ol class="questions">
+                  <li class="group">
+                    <ol class="questions compact">
+                      <li>
+                        <label for="date">
+                          <span class="label">Date</span>
+                          <abbr class="required" title="(required)">*</abbr>
+
+                        </label>
+                        <input id="date" name="date" size="10" required="required"
+                               type="text" />
+                      </li>
+                      <li>
+                        <label for="time">
+                          <span class="label">Time</span>
+                          <abbr class="required" title="(required)">*</abbr>
+
+                        </label>
+                        <input id="time" name="time" size="10" required="required"
+                               type="text" />
+                      </li>
+                    </ol>
+                  </li>
+                  <li>
+                    <label for="csa-s-name">
+                      <span class="label">CSAs name</span>
```

Comment:
Needs an apostrophe in `CSA's`.

Reply:
I have fixed this, please check the latest commit